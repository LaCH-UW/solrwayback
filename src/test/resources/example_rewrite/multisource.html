<html>
<head>
    <title>Some page</title>
    <link rel="stylesheet" type="text/css" href="styles/index.css" />
</head>
<h1>srcset testing</h1>

<p>An image with absolute srcset: <img src="http://example.com/foo.jpg" srcset="http://example.com/fooF.jpg 1x, http://example.com/fooF2.jpg 2x"/></p>
<p>An image with absolute srcset and substring trickery: <img src="http://example.com/foo.jpg" srcset="http://example.com/foo.jpg 1x, http://example.com/foo.jpg.bar.jpg 2x"/></p>
<p>An image with relative srcset: <img src="fooB.jpg" srcset="fooB.jpg 1x, fooB2.jpg 2x"/></p>
<p>An image with relative srcset and substring trickery: <img src="fooS.jpg" srcset="fooS.jpg 1x, barfooS.jpg 2x"/></p>
<p>An image with root srcset: <img src="/fooC.jpg" srcset="/fooC.jpg 1x, /fooC2.jpg 2x"/></p>

</html>
