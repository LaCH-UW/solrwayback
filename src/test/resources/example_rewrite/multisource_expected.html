<html>
<head>
    <title>Some page</title>
    <link rel="stylesheet" type="text/css" href="styles/index.css" />
</head>
<h1>A very simple page</h1>

<p>An <a href="http://example.com">absolute link</a> leading to example.com</p>
<p>A <a href="subfolder/">relative link</a> leading to <tt>subfolder</tt>subfolder</tt></p>
<p>A <a href="/otherpage.html">root-based link</a> leading to a parallel page <tt>otherpage.html</tt>.</p>

<p>An absolute referenced image: <img src="http://example.com/image.jpg"/></p>
<p>A relatively referenced image: <img src="subfolder/image.jpg"/></p>
<p>A root referenced image: <img src="/image.jpg"/></p>

<p>An image with absolute srcset: <img src="http://example.com/foo.jpg" srcset="http://example.com/fooF.jpg 1x, http://example.com/fooF2.jpg 2x"/></p>
<p>An image with absolute srcset and substring trickery: <img src="http://example.com/foo.jpg" srcset="http://example.com/foo.jpg 1x, http://example.com/foo.jpg.bar.jpg 2x"/></p>
<p>An image with relative srcset: <img src="fooB.jpg" srcset="fooB.jpg 1x, fooB2.jpg 2x"/></p>
<p>An image with relative srcset and substring trickery: <img src="fooS.jpg" srcset="fooS.jpg 1x, barfooS.jpg 2x"/></p>
<p>An image with root srcset: <img src="/fooC.jpg" srcset="/fooC.jpg 1x, /fooC2.jpg 2x"/></p>

</html>
